lib/records/I_RecCore.h

```diff
@@ -138,8 +138,14 @@ RecErrT RecRegisterRawStatUpdateFunc(const char *name, RecRawStatBlock *rsb, int
 // already been taken out for the callback.
 
 // RecSetRecordConvert -> WebMgmtUtils.cc::varSetFromStr()
-RecErrT RecSetRecordConvert(const char *name, const RecString rec_string, RecSourceT source, bool lock = true,
+RecErrT RecSetRecordConvert(std::string_view name, const RecString rec_string, RecSourceT source, bool lock = true,
                             bool inc_version = true);
+inline RecErrT
+RecSetRecordConvert(const char *name, const RecString rec_string, RecSourceT source, bool lock = true, bool inc_version = true)
+{
+  return RecSetRecordConvert(std::string_view{name ? name : ""}, rec_string, source, lock, inc_version);
+}
+
```

Comment:
Should we maybe just rely on implicit conversion from from C strings to string_views?

http://coliru.stacked-crooked.com/a/e5b6e9246874639c

Reply:
In this situation, you would get either ambiguous overloads, or infinite recursion. Also, as you can see, this will crash on `nullptr` if we depend on implicit converion (with clang, it does work with gcc, but that's depending on undefined behavior). This is one of the points of disagreement between me and the standards committee.